internal/donor/donorpage/delete_lpa.go

```diff
@@ -20,9 +20,24 @@ type deleteLpaData struct {
 	Donor  *donordata.Provided
 }
 
-func DeleteLpa(tmpl template.Template, donorStore DonorStore, notifyClient NotifyClient) Handler {
+func DeleteLpa(tmpl template.Template, donorStore DonorStore, notifyClient NotifyClient, appPublicURL string) Handler {
 	return func(appData appcontext.Data, w http.ResponseWriter, r *http.Request, provided *donordata.Provided) error {
 		if r.Method == http.MethodPost {
+			if !provided.CertificateProviderInvitedAt.IsZero() {
+				email := notify.InformCertificateProviderLPAHasBeenDeleted{
+					DonorFullName:                   provided.Donor.FullName(),
+					DonorFullNamePossessive:         appData.Localizer.Possessive(provided.Donor.FullName()),
+					LpaType:                         localize.LowerFirst(appData.Localizer.T(provided.Type.String())),
+					CertificateProviderFullName:     provided.CertificateProvider.FullName(),
+					InvitedDate:                     appData.Localizer.FormatDate(provided.CertificateProviderInvitedAt),
+					CertificateProviderStartPageURL: appPublicURL + appData.Lang.URL(page.PathCertificateProviderStart.Format()),
```

Comment:
seeing this has just made me realise I missed this on the emails I did :smile:...